### PR TITLE
Setup New Features

### PR DESCRIPTION
**Pull request by okemboi**
In this branch, I did;
[ ] Edit ReadMe file
[ ] Add HTML file
[ ] Add CSS file 
[ ] Add Linters

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (Shedrack-Sunday) [inline .gitignore:1]: - [ ] Part of the project requirements was for you to add a `test.md` file, and make sure it's ignored by git. kindly follow the instructions below to do this 👍 .
- The purpose of the `.gitignore` is to take away abstracted files, and leave your project lightweight. If you attempt to comment the `node_modules` folder here, you'll see thousands of javascript support files which aid your project but are abstract to not be added here. the instructions here is to teach you to do so 👍 

## Instrctions 
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @okemboi       👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 

  
# Required Changes ♻️

- [x] I must admit you have really put in the effort so far in this project, however, the changes required below need to be implemented :


_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: .gitignore (hunk 1 above)
Comment (levy002) [inline .gitignore:1]: - [ ] I can see that the `test.md` file is not presented in the repository which is correct but as required the file must be ignored by git so it means it must be presented in the `gitignore` file. Kindly make sure the `test.md` is presented in the `gitignore` file in order to meet the project requirement. Kindly fix that as suggested by the previous reviewer.

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @okemboi   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Good commit history :heavy_check_mark: 
- Professional README :heavy_check_mark: 
- Linters are green :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by osamaashraf6 — APPROVED** [review]
Hi @okemboi 😊, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
Congratulations!GIF

## Highlights🎉
- No linter errors present
- The PR has a proper title and a descriptive body 👍

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by okemboi** [comment]
It's merged. Do I need to make another submission in my dashboard or its good as is?

**Review by levy002 — COMMENTED** [review]
## Hello @okemboi 

Your project is complete!  There is nothing else to say other than...  Congratulations! 🎉 🎉 🎉 

## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by levy002** [comment]
> It's merged. Do I need to make another submission in my dashboard or its good as is?

Hello @okemboi , because the project was solo just go to your dashboard and submit normally your project is approved!

**Comment by okemboi** [comment]
Done. Thank you

**Pull request merged**